calico-vpp-agent/cmd/calico_vpp_dataplane.go

```diff
@@ -72,22 +74,22 @@ func main() {
 		return
 	}
 
-	routingServer, err := routing.NewServer(vpp, log.WithFields(logrus.Fields{"component": "routing"}))
+	routingServer, err := routing.NewServer(vpp, log.WithFields(logrus.Fields{"component": "routing"}), &mainTomb)
```

Comment:
In all the servers it looks like we only use the Dying() channel of the main Tomb. It would be cleaner to pass just that channel.